### PR TITLE
Add categories in update project

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: api-client/resources/projects/adapters.ts [hunk 1]
```diff
@@ -10,6 +11,10 @@ const Adapters: { [key in Paths]: FetchAdapaterConstructor } = {
     url: "projects",
     method: "GET",
   },
+  get_all_categories: {
+    url: "project-categories",
```
Comment (haydencleary) [inline api-client/resources/projects/adapters.ts:15]: Shouldn't this be inside a different resource ?

File: api-client/resources/projects/fetch/get-all-projects-categories.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { FetchAdapter } from "api-client/adapter/fetch/fetch-adapter";
+import { IFetchAdapater } from "api-client/adapter/fetch/fetch-adapter.types";
+import { ParametersInterface } from "api-client/types/parameters-interface";
+
+import adapters from "../adapters";
+import tags from "../tags";
+import { GetProjectCategories } from "../types";
+
+export function getAllProjectsCategories(_: ParametersInterface<object>): IFetchAdapater<GetProjectCategories> {
```
Comment (haydencleary) [inline api-client/resources/projects/fetch/get-all-projects-categories.ts:9]: Why don't you remove the parameter as it's not used ?

File: api-client/resources/projects/adapters.ts (hunk 1 above)
Comment (alexbeno) [reply api-client/resources/projects/adapters.ts:15]: when we suggest this we have a route /project/categories same for create and update 

so i think for us the ressource is project

File: api-client/resources/projects/fetch/get-all-projects-categories.ts (hunk 2 above)
Comment (alexbeno) [reply api-client/resources/projects/fetch/get-all-projects-categories.ts:9]: to use the typeof inside the react query, if we remove this, we will have to refactor to extends the route

**Review by haydencleary — APPROVED** [review]

**Pull request merged**